app/helpers/room_helper.rb

```diff
@@ -16,23 +16,24 @@ def done_los(results,lo)
 
   # Vytvori a vrati popis danej miestnosti na zaklade najpouzivanejsich konceptov
   def get_description(room)
-    los = room.learning_objects.eager_load(:concepts)
-    description = "Najviac zastúpené témy: "
-    list = Hash.new
+    lo = room.learning_objects.shuffle.first
+    description = "Zastúpené témy: "
+    list = Array.new
 
-    los.each do |l|
-      l.concepts.each do |c|
-        if list.has_key?(c.name)
-          list[c.name] = list[c.name] + 1
-        else
-          list[c.name] = 1
+    concepts = lo.concepts
+    concepts.each do |c|
+        if list.count < Room::NUMBER_OF_CONCEPTS_IN_DESCRIPTION
+          list.push(c.name)
         end
+      if list.count == Room::NUMBER_OF_CONCEPTS_IN_DESCRIPTION
+        break
       end
     end
 
-    list = list.sort_by{|_key, value| value}.reverse.to_h
-    description += list.take(Room::NUMBER_OF_CONCEPTS_IN_DESCRIPTION).map{ |key, value| "#{key} => #{value}-krat"}.join(', ') + '.'
+    list.each do |li|
+      description += "#{li}, "
+    end
 
-    description
-  end
+    "#{description[0..description.size-3]}."
```

Comment:
Toto ak ma vymazat ciarku a medzeru za poslednym prvkom, tak je to hrozny postup, cele toto aj s tym cyklom s pridavanim ciarky vyssie staci napisat takto nejako:
`"#{list.join(", ")}."